docs/Actions/send-email.md

```diff
@@ -0,0 +1,531 @@
+                    "Settings": {
+                        "Group": "Context"
+                    }
+                }
+            },
+            {
+                "Title": "Send Email",
+                "ActionType": "SendMail",
+                "Description": "Complex example",
+                "Condition": null,
+                "Parameters": {
+                    "From": "<The Sender of the Email>support@plantanapp.com",
+                    "DetermineEmail": "",
+                    "To": "user1@plantanapp.com; user2@plantanapp.com; [User:Email]",
+                    "ToAllUsers": "",
+                    "ReplyTo": "",
+                    "Cc": "user3@plantanapp.com; user4@plantanapp.com;",
+                    "Bcc": "user5@plantanapp.com; user6@plantanapp.com;",
+                    "Subject": "Multiple Email sent",
+                    "Format": {
+                        "Expression": "",
+                        "Value": "Html",
+                        "IsExpression": false,
+                        "Parameters": {}
+                    },
+                    "DnnEmailTemplate": {
+                        "Expression": "",
+                        "Value": "",
+                        "IsExpression": false,
+                        "Parameters": {}
+                    },
+                    "Body": "<p>Dear Customers,</p><p>This is an example using tokens. The email was sent to&nbsp;</p>",
+                    "Headers": "",
+                    "PortalFiles": "",
+                    "FileField1": {
+                        "Expression": "",
+                        "Value": "",
+                        "IsExpression": false
+                    },
+                    "FileField2": {
+                        "Expression": "",
+                        "Value": "",
+                        "IsExpression": false
+                    },
+                    "FileField3": {
+                        "Expression": "",
+                        "Value": "",
+                        "IsExpression": false
+                    },
+                    "FileField4": {
+                        "Expression": "",
+                        "Value": "",
+                        "IsExpression": false
+                    },
+                    "FileField5": {
+                        "Expression": "",
+                        "Value": "",
+                        "IsExpression": false
+                    },
+                    "FileToken1": "[Path]",
+                    "IgnoreErros": ""
+                },
+                "$_uid": "action15918762609042327",
+                "$_isOpen": false,
+                "$_isLoaded": true,
+                "$_isFocus": true
+            }
+        ]
+    }
+}
+```
+
+### `4. Sending an email with Load User from SQL action help`
+
+In this example will load all the users in the system and send them an email. We are using the `Send mail to all users` parameter instead of setting the `To` parameter.

```

Comment:
```suggestion
In this example, we will load all the existing users directly from the database and send them an email. Note that we are using the `Send mail to all users` parameter instead of setting the `To` parameter.

```